api/api_autogen/library/defaults/Cashloan_PVBatteryResidential.json

```diff
@@ -16,23 +16,23 @@
 			"real_discount_rate": 6.4,
 			"salvage_percentage": 0,
 			"state_tax_rate": [7.000000],
-			"system_capacity": 7.03931
+			"system_capacity": 8.00718
 		},
 		"SystemCosts": {
 			"add_om_num_types": 1,
-			"om_batt_capacity_cost": [9.680000],
+			"om_batt_capacity_cost": [14.810000],
```

Comment:
Difference in precision between this and standalone battery residential (15)? Custom battery residential appears to have 14.81 as well